src/test/java/com/thebois/models/world/StructureTest.java

```diff
@@ -0,0 +1,40 @@
+package com.thebois.models.world;
+
+import org.junit.jupiter.api.Test;
+
+import com.thebois.models.Position;
+import com.thebois.models.world.structures.Structure;
+import com.thebois.models.world.structures.StructureType;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class StructureTest {
+
+    @Test
+    public void getPositionTest() {
```

Comment:
Just some of my opinions regarding test naming:

* There's no point in having the word "test" in the name, since all the methods in the class are tests.
* The test name should describe *which* method is being tested (getPosition) and *what is being confirmed* (ReturnsPositionSetInConstructor) for a final name of `getPositionReturnsPositionSetInConstructor`.
* There is no such thing as a test with too long a name.

Reply:
Will take a look at the tests and add the missing ones and give them proper names.

Reply:
Another thing - I think we've been calling all our test classes `xxxxTests` and not `xxxxTest` so far. Best to follow that as well.